src/pages/workspace/WorkspacePeoplePage.js

```diff
@@ -0,0 +1,328 @@
+import {withOnyx} from 'react-native-onyx';
+import styles from '../../styles/styles';
+import ONYXKEYS from '../../ONYXKEYS';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Navigation from '../../libs/Navigation/Navigation';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import compose from '../../libs/compose';
+import {flatListRef} from '../../libs/ReportScrollManager';
+import {removeMembers} from '../../libs/actions/Policy';
+import Avatar from '../../components/Avatar';
+import Button from '../../components/Button';
+import Checkbox from '../../components/Checkbox';
+import Text from '../../components/Text';
+import ROUTES from '../../ROUTES';
+import ConfirmModal from '../../components/ConfirmModal';
+import personalDetailsPropType from '../personalDetailsPropType';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../../components/withWindowDimensions';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+
+    ...windowDimensionsPropTypes,
+
+    /** The personal details of the person who is logged in */
+    personalDetails: personalDetailsPropType.isRequired,
+
+    /** The policy passed via the route */
+    policy: PropTypes.shape({
+        /** The policy name */
+        name: PropTypes.string,
+    }),
+
+    /** URL Route params */
+    route: PropTypes.shape({
+        /** Params from the URL path */
+        params: PropTypes.shape({
+            /** policyID passed via route: /workspace/:policyID/people */
+            policyID: PropTypes.string,
+        }),
+    }).isRequired,
+};
+
+const defaultProps = {
+    policy: {
+        name: '',
+    },
+};
+
+class WorkspacePeoplePage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            selectedEmployees: [],
+            isRemoveMembersConfirmModalVisible: false,
+        };
+
+        this.inviteUser = this.inviteUser.bind(this);
+        this.removeUsers = this.removeUsers.bind(this);
+        this.toggleUser = this.toggleUser.bind(this);
+        this.renderItem = this.renderItem.bind(this);
+        this.renderHeader = this.renderHeader.bind(this);
+        this.askForConfirmationToRemove = this.askForConfirmationToRemove.bind(this);
+        this.hideConfirmModal = this.hideConfirmModal.bind(this);
+    }
+
+    /**
+    * Open the modal to invite a user
+    */
+    inviteUser() {
+        Navigation.navigate(ROUTES.getWorkspaceInviteRoute(this.props.route.params.policyID));
+    }
+
+    /**
+    * Remove selected users from the workspace
+    */
+    removeUsers() {
+        // Remove the admin from the list
+        // eslint-disable-next-line max-len
+        removeMembers(this.state.selectedEmployees.filter(email => email !== this.props.session.email), this.props.route.params.policyID);
+        this.setState(prevState => ({
+            ...prevState,
```

Comment:
There's no need to spread `prevState` in here react will know that you only want to modify the state values specified.

Reply:
And since we are not referencing the `prevState` we can call `this.setState({someProperty: value});`

Reply:
I have been spreading the prevState unnecessarily for last 2 years 🙃 thanks for heads up!